vm/runtime.ts

```diff
@@ -1467,12 +1482,8 @@ module J2ME {
       }
     }
 
-    if (false && methodTimeline) {
-      fn = profilingWrapper(fn, methodInfo, methodType);
-    }
-
-    if (traceWriter) {
-      fn = tracingWrapper(fn, methodInfo, methodType);
+    if (profile) {
+      fn = wrapMethod(fn, methodInfo, methodType);
```

Comment:
I'm ok with that. The trace writer currently outputs more readable graphs, but we can fix that in the profiler.

I've also been wanting an easy way to enable/disable the trace writer at runtime, so I can trace only particular interactions. That should be doable either way, though.